ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dot1x.j2

```diff
@@ -39,6 +39,7 @@
 {%             set reauthentication = ethernet_interface.dot1x.reauthentication | arista.avd.default('-') %}
 {%             set host_mode = ethernet_interface.dot1x.host_mode.mode | arista.avd.default('-') %}
 {%             set mac_based_authentication_enabled = ethernet_interface.dot1x.mac_based_authentication.enabled | arista.avd.default('-') %}
+{%             set auth_failure_fallback_mba = ethernet_interface.dot1x.eapol.authentication_failure_fallback_mba.enabled | arista.avd.default('-') %}
```

Comment:
Is this the value you want to show? Not an inverted `dot1x.eapol.disabled`? Also you are not showing the value at all, since you don't include it in the line below.
Make sure the title in the table corresponds with the field.

Reply:
Fixed